### PR TITLE
{tools}[foss/2019b] MEM v20191023

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)
requires ~~#10585~~ (required R extensions)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, x86_64, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/f77005c7d9db7a31e608e1f628c190de for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3125.skitty.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/0d7b54a6ba4a2501859313446e61ea4c for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2658.swalot.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 2.7.5
See https://gist.github.com/65d7745a67bffd0ff2092a746bf8312e for a full test report.

File: easybuild/easyconfigs/m/MEM/MEM-20191023-foss-2019b-R-3.6.2.eb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+easyblock = 'RPackage'
+
+name = 'MEM'
+local_commit = '6b92476'
+version = '20191023'
+versionsuffix = '-R-%(rver)s'
```
Comment (smoors) [inline easybuild/easyconfigs/m/MEM/MEM-20191023-foss-2019b-R-3.6.2.eb:6]: @boegel I thought this was not necessary, as R-3.6.2 is the default for 2019b?
apparently some easyconfigs have added it, and other haven't...

File: easybuild/easyconfigs/m/MEM/MEM-20191023-foss-2019b-R-3.6.2.eb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+source_urls = ['https://github.com/cytolab/mem/archive/']
+sources = [{'download_filename': '%s.tar.gz' % local_commit, 'filename': SOURCE_TAR_GZ}]
+checksums = ['735cc340fb644ddc4bd760d6897d3911561ea4735db10e799ff34e36ba138a86']
+
+dependencies = [
+    ('R', '3.6.2'),
+    ('R-bundle-Bioconductor', '3.10'),
+]
+
+modextrapaths = {'R_LIBS': ''}
```
Comment (smoors) [inline easybuild/easyconfigs/m/MEM/MEM-20191023-foss-2019b-R-3.6.2.eb:22]: @boegel this is done automatically for RPackage, so can be removed.

File: easybuild/easyconfigs/m/MEM/MEM-20191023-foss-2019b-R-3.6.2.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/m/MEM/MEM-20191023-foss-2019b-R-3.6.2.eb:6]: Yeah, we're not exactly consistent there... I'll get rid of it here.

File: easybuild/easyconfigs/m/MEM/MEM-20191023-foss-2019b-R-3.6.2.eb (hunk 2 above)
Comment (boegel) [reply easybuild/easyconfigs/m/MEM/MEM-20191023-foss-2019b-R-3.6.2.eb:22]: Good point, will clean up.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, x86_64, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/1f5ed4989c13860f6821437e99e13003 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3416.kirlia.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz (cascadelake), Python 2.7.5
See https://gist.github.com/e0180f5dbf372d8fa33833b39be6d308 for a full test report.

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node354.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 2.7.5
See https://gist.github.com/f6dcec006ba5b566cacd722593e13b60 for a full test report.

**Comment by smoors** [comment]
Going in, thanks @boegel!

**Pull request merged**